src/status_im/utils/fx.clj

```diff
@@ -26,17 +36,27 @@
         m (if (map? (first fdecl))
             (conj m (first fdecl))
             m)
+        events (get m :events [])
+        interceptors (get m :interceptors [])
         fdecl (if (map? (first fdecl))
                 (next fdecl)
                 fdecl)
         [cofx & args] (first fdecl)
         fdecl (next fdecl)
         argsyms (take (count args) (repeatedly #(gensym "arg")))]
-    `(clojure.core/defn ~(with-meta name m)
-       ([~@argsyms] (fn [cofx#] (~name cofx# ~@argsyms)))
-       ([cofx# ~@args]
-        (if (and (map? cofx#)
-                 (not (nil? (:db cofx#))))
-          (let [~cofx cofx#]
-            ~@fdecl)
-          (throw (js/Error. (str "fx/defn expects a map of cofx as first argument got " cofx# " in function " ~name))))))))
+    (if (and (sequential? events)
+             (every? keyword? events))
+      `(do
+         (clojure.core/defn ~(with-meta name m)
+           ([~@argsyms] (fn [cofx#] (~(with-meta name m) cofx# ~@argsyms)))
+           ([cofx# ~@args]
+            (if (and (map? cofx#)
+                     (not (nil? (:db cofx#))))
+              (let [~cofx cofx#]
+                ~@fdecl)
+              (throw (js/Error. (str "fx/defn expects a map of cofx as first argument got " cofx# " in function " ~name))))))
+         ~(when (not-empty events)
+            `(when-not (find-ns 'status-im.utils.handlers)
```

Comment:
it seems that we don't want to use `find-ns` anymore

Reply:
I don't think it can build without it before your PR is merged

Reply:
do you mean it will work only with `shadow-cljs`?

Reply:
I mean we will have warnings if I remove it now

Reply:
Warning during development or in release builds?

Reply:
I don't see any warnings in dev mode

Reply:
I mean, without `find-ns`

Reply:
that is because `[status-im.utils.handlers :as handlers]` is already required in the only place namespace where the part that needs it is macro-expended.

Reply:
@rasom I removed it can you unblock the PR?